doc/manuals/admin/build_source.md

```diff
@@ -116,17 +116,15 @@ In the case of the aarch64 architecture, install libxslt using yum, and run `./c
         sudo yum install curl nc valgrind bc
         sudo pip2 install virtualenv
 
-In the case of the aarch64 architecture, additionally install python2-devel, rpm-build and libffi-devel using yum. It is needed when building pyOpenSSL.
```

Comment:
No need of extra tooling to installing pyOpenSSLin Python3 in aarch64 architecture?

(Just to be sure to remove this paragraph in that case)